infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -20,7 +23,13 @@
         </encoder>
     </appender>
 
+    <!-- https://logback.qos.ch/manual/appenders.html#AsyncAppender -->
+    <appender name="async_file" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="file" />
+        <queueSize>1024</queueSize>
```

Comment:
Is there a way to tune this parameter?

Reply:
You define this parameter it's up to you. Don't get your question

Reply:
I mean this is hardcoded now, is there any way to change this parameter when you deploy corfu server without having to manually change the file, like this: https://github.com/CorfuDB/CorfuDB/blob/master/infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java#L115

Reply:
You have direct access to `logback-prod.xml` and can/should make changes in that file if needed. I strongly believe this parameter shall not be configured via command line

Reply:
Changing the source file on every deployment is not a good idea. I believe it's better to have this parameter configurable as it shall change in terms of production machine's memory, IO bandwidth, etc. But anyway it's up to you.

Reply:
It must be in an xml config file. There are many proper ways to manage these parameters in/for different environments. Changing it via command-line definitely a wrong way. 
If we would need to make this parameter more dynamic we will figure out that. For now, this is not needed